meson_options.txt

```diff
@@ -66,7 +66,7 @@ option('trim_dsp',
 
 option('test_rust',
     type: 'boolean',
-    value: true,
+    value: false,
```

Comment:
Does this change how we should be running `test.sh` locally?

Reply:
You're passing it the path to the Rust binaries explicitly, right? If so, this shouldn't change anything. It just was defaulting to building the C CLI tools against librav1d.a instead of libdav1d.so.

Reply:
> You're passing it the path to the Rust binaries explicitly, right?

Yeah.